client/src/components/PlaceholderComponent/PlaceholderComponentTitle/PlaceholderComponentTitle.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './PlaceholderComponentTitle.scss';
+
+interface PlaceholderComponentProps {
+    source: string;
+    startTime: string;
+    endTime: string;
+}
+
+const PlaceholderComponentTitle: React.FC<PlaceholderComponentProps> =
+    ({source, startTime, endTime}) => {
+        return (
+            <div className="placeholder-title">
+                <p className="title">
+                    Last report from
+                </p>
+                <a href="#" className="source">{source}</a>
+                <p className="date-range">
+                    ({startTime} - {endTime})
+                </p>
+            </div>
+        );
+    };
+
+export default PlaceholderComponentTitle;
```

Comment:
Consider changing name from placeholder to, for example "Section". IMO placeholder suggests that something is temporary.